core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci

```diff
@@ -16,3 +16,4 @@ quay.io/redhatqe/insights-operator-tests:4.7 registry.ci.openshift.org/ci/insigh
 quay.io/openshift-cr/job-run-aggregator:latest registry.ci.openshift.org/ci/job-run-aggregator:latest
 quay.io/opdev/preflight:stable registry.ci.openshift.org/ci/openshift-preflight:latest
 registry.fedoraproject.org/fedora:latest registry.ci.openshift.org/ci/fedora:latest
+docker.io/golangci/golangci-lint:v1.43.0 registry.ci.openshift.org/edge-infrastructure/golangci-lint:v1.43.0
```

Comment:
To `ci` namespace please, where the other `golangci-lint` images currently are. There's no reason why they should be scattered over the cluster.

Reply:
Also, maybe we have a `quay.io` source for this? Dockerhub sources are problematic because of their strict rate limits.

Reply:
Also^2, I'm surprised I don't see the other golangci-lint images in this list - as we are touching this file, do you think you can also add the two versions from https://github.com/openshift/release/blob/master/clusters/app.ci/supplemental-ci-images/golangci-lint_is.yaml ?

It looks like these images were kinda forgotten about :(

Reply:
The only ref I see for other `golangci-images` comes from `clusters/app.ci/supplemental-ci-images/golangci-lint_is.yaml` and you requested to remove it [here](https://github.com/openshift/release/pull/24508#discussion_r767598772)

Unless I am missing something @petr-muller

Reply:
Yeah, so what I think we need:

1. Add all three lines to `core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci`: one for the version you are trying to add, and two for the versions from the imagestream (which I would expect to be here alreaady - this is not your fault, but are in a position to fix it now)
2. Find a quay.io source for the images - looks liie our SREs are mirroring the images to https://quay.io/repository/app-sre/golangci-lint so we could use that
3. All these imafges should go to the `ci` namespace

Reply:
Fixed.